sql/src/main/java/io/crate/protocols/postgres/ConnectionContext.java

```diff
@@ -158,6 +158,7 @@
 class ConnectionContext {
 
     private static final Logger LOGGER = Loggers.getLogger(ConnectionContext.class);
+    private static final String HBA_PROTOCOL = "pg";
```

Comment:
`POSTGRES_PROTOCOL_NAME`

I would move the constant into `AuthenticationProvider` class